### PR TITLE
SAK-30715: Use mail.support for boxes 2 and 3 if technicalAddress not set

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

File: feedback/src/java/org/sakaiproject/feedback/tool/FeedbackTool.java [hunk 1]
```diff
@@ -152,6 +152,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
             siteEmail = site.getProperties().getProperty(Site.PROP_SITE_CONTACT_EMAIL);
         }
 
+        String mailSupport = sakaiProxy.getConfigString("mail.support", "");
```
Comment (ottenhoff) [inline feedback/src/java/org/sakaiproject/feedback/tool/FeedbackTool.java]: Can you use a CONSTANT here instead?

File: feedback/src/webapp/js/feedback.js [hunk 2]
```diff
@@ -82,7 +83,7 @@
                     $('#feedback-technical-setup-instruction').show();
                 }
 
-                if (feedback.supplementaryInfo.length > 0) {
+                if (feedback.supplementaryInfo.length == 0 && feedback.mailSupport==null) {
```
Comment (ottenhoff) [inline feedback/src/webapp/js/feedback.js]: This seems to change the previous logic ..... is it correct?

Comment (nicholaswilson100) [inline feedback/src/webapp/js/feedback.js]: Yes, the logic before said (incorrectly) to show the message box (saying that you need to set a property) if the property was set - which was not right.  

Now it says show the message box if neither of 2 properties is set ('feedback.supplementaryInfo, mail.support) (if you set either then you get some functionality).

File: feedback/src/java/org/sakaiproject/feedback/tool/FeedbackTool.java [hunk 3]
```diff
@@ -152,6 +152,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
             siteEmail = site.getProperties().getProperty(Site.PROP_SITE_CONTACT_EMAIL);
         }
 
+        String mailSupport = sakaiProxy.getConfigString(Constants.MAIL_SUPPORT, "");
```
Comment (ottenhoff) [inline feedback/src/java/org/sakaiproject/feedback/tool/FeedbackTool.java]: Should this property have a default value?

**Comment by nicholaswilson100** [comment]
I've rebased the request so it's a clean merge and given the same default value to 'mail.support' as in other places (e.g., emailtemplateservice).

**Comment by ottenhoff** [comment]
We can't auto-merge this into master... can you rebase again?

**Comment by nicholaswilson100** [comment]
Thanks Sam I've rebased this - I wish there was some email notification (maybe there is but I don't get it) that tells you when your code becomes an unclean merge.

**Comment by nicholaswilson100** [comment]
Then the Sakai mergers wouldn't have to copy and paste the line 'Can you rebase this please?' for some of the time...

**Pull request merged**